### PR TITLE
Fixed UI issues

### PR DESCRIPTION
**Pull request by Sahil1479**
- Resolved following UI related issues.

image
- Added label (in header) to the table for Memory and CPU tabs.

### CODE REVIEW

File: web/pgadmin/dashboard/static/js/SystemStats/Storage.jsx [hunk 1]
```diff
@@ -483,7 +483,7 @@ export function StorageWrapper(props) {
                 ],
               }}
               options={
-                {
+                {
```
Comment (khushboovashi) [inline web/pgadmin/dashboard/static/js/SystemStats/Storage.jsx:486]:
```suggestion
                {
```

File: web/pgadmin/static/js/components/PgChart/StreamingChart.jsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ function tooltipPlugin(refreshRate) {
       document.body.appendChild(tooltip);
     }
   }
-
+
```
Comment (khushboovashi) [inline web/pgadmin/static/js/components/PgChart/StreamingChart.jsx:32]:
```suggestion
 
```

**Review by khushboovashi — CHANGES_REQUESTED** [review]

**Comment by Sahil1479** [comment]
@khushboovashi As suggested, I've removed the trailing white spaces.

**Comment by khushboovashi** [comment]
> This branch is out-of-date with the base branch

Please rebase your branch, so I can merge the changes.

**Comment by akshay-joshi** [comment]
@Sahil1479 

Can you please rebase the branch and close all the conversations asap?

**Comment by Sahil1479** [comment]
Hi @khushboovashi and @akshay-joshi, actually I'm out of station and don't have laptop with me. I'll do it by EOD. Is that fine?

**Comment by akshay-joshi** [comment]
> Hi @khushboovashi and @akshay-joshi, actually I'm out of station and don't have laptop with me. I'll do it by EOD. Is that fine?

Yes, no problem

**Comment by Sahil1479** [comment]
@khushboovashi, @akshay-joshi Rebased the branch and closed all the conversations.

**Pull request merged**